regression/cbmc/union13/main.c

```diff
@@ -0,0 +1,11 @@
+#include <assert.h>
+
+int main()
+{
+  union {
+    unsigned long long i;
```

Comment:
AFAIK `unsigned long` is the type guaranteed to have same size as a pointer

Reply:
Not on Windows/LLP64...

Reply:
Gah, so it isn't, Though perhaps `intptr_t` is better?

Reply:
Done.